NugetLightProvider.cs

```diff
@@ -137,7 +137,7 @@ public void GetDynamicOptions(string category, Request request)
                     request.YieldDynamicOption("Destination", Constants.OptionType.Folder, false);
                     request.YieldDynamicOption("ExcludeVersion", Constants.OptionType.Switch, false);
                     request.YieldDynamicOption("Scope", Constants.OptionType.String, false, new[] { Constants.CurrentUser, Constants.AllUsers });
-                    //request.YieldDynamicOption("SkipDependencies", Constants.OptionType.Switch, false);
+                    request.YieldDynamicOption("SkipDependencyCheck", Constants.OptionType.Switch, false);
```

Comment:
I prefer SkipDependencies because it clearly means skip installation of dependencies